src/sgemm_kernel.rs

```diff
@@ -295,6 +306,10 @@ unsafe fn kernel_x86_avx(k: usize, alpha: T, a: *const T, b: *const T,
     ab[6] = ab6666;
     ab[7] = ab7777;
 
+    macro_rules! c {
+        ($i:expr, $j:expr) => (c.offset(rsc * $i as isize + csc * $j as isize));
+    }
+
```

Comment:
duplicated macro def

Reply:
Forgot to force-push my fix for that last night.